### PR TITLE
Fixed serialize_example() example description and applied nbfmt 

### PR DESCRIPTION
**Pull request by eslam69**
This is a follow-up pull request regarding #2198, as @markmcd asked, I've run nbfmt on the notebook and passed nblint test locally.
- Fixed serialize_example() example description where the mentioned function `create_message()` was meant to be `serialize_example()`.
- Updated the code cell where the `serialize_example()` is used, the `example_observation` variable was initialized but not used, so I filled it with the data, and used it as an argument to  `serialize_example()`.

### CODE REVIEW

File: site/en/tutorials/load_data/tfrecord.ipynb [hunk 1]
```diff
@@ -350,27 +372,30 @@
...
     },
     {
+      "attachments": {},
       "cell_type": "markdown",
       "metadata": {
         "id": "XftzX9CN_uGT"
       },
       "source": [
-        "For example, suppose you have a single observation from the dataset, `[False, 4, bytes('goat'), 0.9876]`. You can create and print the `tf.train.Example` message for this observation using `create_message()`. Each single observation will be written as a `Features` message as per the above. Note that the `tf.train.Example` [message](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/example/example.proto#L88) is just a wrapper around the `Features` message:"
+        "For example, suppose you have a single observation from the dataset, `[False, 4, bytes('goat'), 0.9876]`. You can create and print the `tf.train.Example` message for this observation using `serialize_example()`. Each single observation will be written as a `Features` message as per the above. Note that the `tf.train.Example` [message](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/example/example.proto#L88) is just a wrapper around the `Features` message:"
```
Comment (8bitmp3) [inline site/en/tutorials/load_data/tfrecord.ipynb:381]: @MarkDaoust @markmcd

File: site/en/tutorials/load_data/tfrecord.ipynb [hunk 2]
```diff
@@ -44,10 +48,10 @@
         "    <a target=\"_blank\" href=\"https://www.tensorflow.org/tutorials/load_data/tfrecord\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />View on TensorFlow.org</a>\n",
         "  </td>\n",
         "  <td>\n",
-        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/tutorials/load_data/tfrecord.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/eslam69/docs/blob/master/site/en/tutorials/load_data/tfrecord.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
```
Comment (MarkDaoust) [inline site/en/tutorials/load_data/tfrecord.ipynb:51]: Please revert this line.

File: site/en/tutorials/load_data/tfrecord.ipynb [hunk 3]
```diff
@@ -44,10 +48,10 @@
...
         "  </td>\n",
         "  <td>\n",
-        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/en/tutorials/load_data/tfrecord.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/eslam69/docs/blob/master/site/en/tutorials/load_data/tfrecord.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
```
Comment (MarkDaoust) [inline site/en/tutorials/load_data/tfrecord.ipynb:54]: Please revert this line.

Use the "open in colab" extension (or a quick bookmarklet) to open your working copy ion colab.

**Review by MarkDaoust — CHANGES_REQUESTED** [review]

File: site/en/tutorials/load_data/tfrecord.ipynb (hunk 3 above)
Comment (eslam69) [reply site/en/tutorials/load_data/tfrecord.ipynb]: Done!

File: site/en/tutorials/load_data/tfrecord.ipynb (hunk 2 above)
Comment (eslam69) [reply site/en/tutorials/load_data/tfrecord.ipynb]: Done!

File: site/en/tutorials/load_data/tfrecord.ipynb [hunk 4]
```diff
@@ -44,10 +48,10 @@
         "    <a target=\"_blank\" href=\"https://www.tensorflow.org/tutorials/load_data/tfrecord\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />View on TensorFlow.org</a>\n",
         "  </td>\n",
         "  <td>\n",
-        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/tutorials/load_data/tfrecord.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/eslam69/docs/blob/master/site/en/tutorials/load_data/tfrecord.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
         "  </td>\n",
         "  <td>\n",
-        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/en/tutorials/load_data/tfrecord.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
```
[the remark this answers is not in the document]
Comment (eslam69) [reply site/en/tutorials/load_data/tfrecord.ipynb]: I've Fixed the links, metadata and formatted the notebook, please check my latest commits.

[the remark this answers is not in the document]
Comment (8bitmp3) [reply site/en/tutorials/load_data/tfrecord.ipynb]: Thanks

**Comment by 8bitmp3** [comment]
@MarkDaoust

**Review by eslam69 — COMMENTED** [review]
Handled the merge conflict.

**Comment by eslam69** [comment]
@MarkDaoust Hello Mark, could you please review my work on the requested changes?

**Comment by 8bitmp3** [comment]
@MarkDaoust @markmcd

**Review by MarkDaoust — APPROVED** [review]

**Pull request closed without merging**